config/standard.yaml

```diff
@@ -207,3 +207,34 @@ benchmarks:
       --device: auto
       --n-trials: '10' # Default: 500
       --n-jobs: '1'
+
+  accelerate:
+    <<<: *defaults
+
+    install_group: torch
+
+    plan:
+      method: njobs
+      n: 1
+
+    # This is for single-node
+    manager_addr: "127.0.0.1"
+    manager_port: 10000
+
+    num_processes: 2
```

Comment:
We (probably) want 8 gpus for the 2.7b model.  And even then it's kind of tight.

Reply:
It runs on the A100 with these settings, I set it to two because I only allocated two for development and it doesn't appear to automatically scale. That is one thing we should fix (use all available GPUs instead of hardcoding num_processes).